Two-body-scf-guess.py

```diff
@@ -0,0 +1,93 @@
+from wrap_pyscf import *
+import numpy as np
+
+mol = [("O", -0.167787, 1.645761, 0.108747),
+       ("H", 0.613411, 1.102620, 0.113724),
+       ("H", -0.093821, 2.209720, -0.643619),
+       ("O", 1.517569, -0.667424, -0.080674),
+       ("H", 1.989645, -1.098799, 0.612047),
+       ("H", 0.668397, -1.091798, -0.139744),
+       ("O", -1.350388, -0.964879, -0.092208),
+       ("H", -1.908991, -1.211298, 0.626207),
+       ("H", -1.263787, -0.018107, -0.055536)]
+
+bs = "aug-cc-pvdz"
+monomers = [mol[0:3], mol[3:6], mol[6:9]]
+dimers = [[mol[0:6], (mol[0:3]+mol[6:9])], [mol[0:6], mol[3:9]], [(mol[0:3]+mol[6:9]), mol[3:9]]]
+#dimers = [[(monomers[0], monomers[1]), (monomers[0], monomers[2]), (monomers[0], monomers[3])], 
+          #[(monomers[0], monomers[1]), (monomers[1], monomers[2]), (monomers[1], monomers[3])],
+          #[(monomers[0], monomers[2]), (monomers[1], monomers[2]), (monomers[2], monomers[3])],
+          #[(monomers[0], monomers[3]), (monomers[1], monomers[3]), (monomers[2], monomers[3])]]
```

Comment:
I couldn't input dimer geometries this way, as I kept getting errors from the repo.

Reply:
And yes I recognize these are for the water tetramer, which is the structure I tried to work with before switching back to trimer.

Reply:
Try `monomers[0] + monomers[1]`. You're expecting `dimers` to be a a list of lists of lists of tuples. The original syntax is going to give you a list of lists of tuples of tuples.

Reply:
That works, thank you.